addons/prometheusadapter/0.5.x/prometheusadapter-2.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: prometheusadapter
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: prometheusadapter
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.5.0-2"
+    appversion.kubeaddons.mesosphere.io/prometheusadapter: "0.5.0"
+    values.chart.helm.kubeaddons.mesosphere.io/prometheusadapter: "https://raw.githubusercontent.com/helm/charts/db62a6c595bbd9904014083edd0faa14de4096b2/stable/prometheus-adapter/values.yaml"
```

Comment:
TODO: update this once https://github.com/helm/charts/pull/19961 is merged

Reply:
Let's just update prometheusadapter-1.yaml given 1.3 is not yet GA'ed